java/runtime-native/src/main/java/org/ray/format/gcs/ClientTableData.java

```diff
@@ -0,0 +1,79 @@
+package org.ray.format.gcs;
```

Comment:
Don't commit this file.

Reply:
I saw that there are some auto-generated  files like this, like [arg.java](https://github.com/ray-project/ray/blob/master/java/runtime-native/src/main/java/org/ray/spi/impl/Arg.java).
I don't know whether we should commit this file in java worker, or make it built by the step  'build.sh'.

Reply:
I think all auto-generated files shouldn't be committed. 
@salah-man is there any reason why Arg.java was committed? If not, let's remove it as well?

Reply:
Yes, Arg.java could be generated totally, but the file of TaskInfo.java add function manually to avoid encoding and decoding cost, we could find it as the comment in TaskInfo.java.